docs/_posts/gadde5300/2023-04-05-finsum_flant5_base_en.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: model
+title: Financial FLAN-T5 Summarization (Base)
+author: John Snow Labs
+name: finsum_flant5_base
+date: 2023-04-05
+tags: [finance, en, flant5, t5, summarization, licensed, tensorflow]
+task: Summarization
+language: en
+edition: Finance NLP 1.0.0
+spark_version: 3.0
+supported: true
+engine: tensorflow
+annotator: FinanceSummarizer
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+FLAN-T5 is a state-of-the-art language model developed by Facebook AI that utilizes the T5 architecture for text summarization tasks. It is trained on a large dataset of diverse texts and can generate high-quality summaries of articles, documents, and other text-based inputs.
+
+## Predicted Entities
+
+
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/finance/models/finsum_flant5_base_en_1.0.0_3.0_1680700696220.zip){:.button.button-orange}
+[Copy S3 URI](s3://auxdata.johnsnowlabs.com/finance/models/finsum_flant5_base_en_1.0.0_3.0_1680700696220.zip){:.button.button-orange.button-orange-trans.button-icon.button-copy-s3}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+document_assembler = nlp.DocumentAssembler()\
+    .setInputCol("text")\
+    .setOutputCol("documents")
+
+flant5 = finance.FinanceSummarizer().pretrained('finsum_flant5_base','en','finance/models')\
```

Comment:
The class in scala will be FinanceSummarizer(), but in python will be Summarizer()